play-validations/memory-footprint/src/main/java/com/google/wear/watchface/dfx/memory/AmbientMemoryFootprintCalculator.java

```diff
@@ -227,22 +186,12 @@ private class Visitor {
         private int numLayers = 0;
 
         /** The number of clocks found so far. */
-        private int numClocks = 0;
+        private int numClocks;
```

Comment:
Was this change intentional?

Reply:
Yes. `numClocks` is passed in the constructor, so setting it here to `0` was redundant.